lms/djangoapps/course_structure_api/v0/serializers.py

```diff
@@ -40,37 +40,3 @@ def get_uri(self, course):
     def get_image_url(self, course):
         """ Get the course image URL """
         return course_image_url(course)
-
-
-class GradingPolicySerializer(serializers.Serializer):
```

Comment:
Is this intended to be in a different PR? I wouldn't expect to see this kind of change here.